roles/openshift_logging_elasticsearch/library/openshift_logging_elasticsearch_facts.py

```diff
@@ -0,0 +1,351 @@
+'''
```

Comment:
We should then move this to either a common role or most likely an installer library